model/team_member.go

```diff
@@ -16,6 +16,12 @@ type TeamMember struct {
 	DeleteAt int64  `json:"delete_at"`
 }
 
+type TeamMemberExtra struct {
+	TeamMember
+	MsgCount     int64 `json:"msg_count"`
+	MentionCount int64 `json:"mention_count"`
+}
```

Comment:
sure, I knew something like this was going to happen, we can discuss this as a team and find a proper solution